README.md

```diff
@@ -1 +1,41 @@
-# Hello_Microverse
\ No newline at end of file
+# Hello_Microverse
+Additional description about the project and its features.
+
+Built With
+HTML AND CSS
+using Visual Studio Code
+Git Bash Terminal 
+
+
+
+Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+👤 Author1
+
+GitHub: @rizwanhasan.learn  
+Twitter: @rizwanhasan21
+LinkedIn: Rizwan Hasan
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Good job making use of the README template provided in the project requirements. However, kindly follow the template structure and only personalize the details in the template.

Your README should follow this structure and spaces as in the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) to make it readable and look more professional.